src/api/Metadata.js

```diff
@@ -28,6 +29,7 @@ import {
     ERROR_CODE_FETCH_METADATA_TEMPLATES,
     ERROR_CODE_FETCH_SKILLS,
     ERROR_CODE_FETCH_METADATA_SUGGESTIONS,
+    ERROR_CODE_EMPTY_METADATA_SUGGESTION,
```

Comment:
```suggestion
    ERROR_CODE_EMPTY_METADATA_SUGGESTIONS,
```
Strange this didnt break before you pushed up